src/toil_scripts/lib/files.py

```diff
@@ -49,6 +53,49 @@ def copy_file_job(job, name, file_id, output_dir):
     copy_files([fpath], output_dir)
 
 
+def copy_to(filename, output_dir, work_dir=None):
+    """
+    Moves files from the working directory to the output directory.
+
+    :param work_dir: the working directory
+    :param output_dir: the output directory
+    :param filenames: remaining arguments are filenames
+    """
+    if os.path.isabs(filename):
+        origin = filename
+        filename = os.path.basename(filename)
+    elif work_dir is None:
+        origin = os.path.abspath(filename)
+    else:
+        origin = os.path.join(work_dir, filename)
+    dest = os.path.join(output_dir, filename)
+    try:
+        shutil.copytree(origin, dest)
+    except OSError as e:
+        if e.errno == errno.ENOTDIR:
```

Comment:
IMO, it'd be preferable to reorder this so that you:

1. check for the existence of the output_dir
2. make the output directory, if it doesn't currently exist
3. then execute the copy

If you reorder like so, I think you can get rid of the try/catch.